tests/system/src/test/java/test/javafx/scene/web/StraightLineTest.java

```diff
@@ -53,6 +53,9 @@
 
 public class StraightLineTest {
     private static final CountDownLatch launchLatch = new CountDownLatch(1);
+    private static final int LINE_THICKNESS = 20;
+    private static final int SKIP_TEXT_BOUNDARY = 32;
```

Comment:
What does this constant represent?

Reply:
LINE_THICKNESS is thickness that is also in html test code. SKIP_TEXT_BOUNDARY is used to skip boundary at the end of line.